lib/diagnostic.rb

```diff
@@ -108,10 +113,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = {favorite_number: 2, first_name: 'Matt'}
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = "#{@person_hash.first_name} is so cool. the best. yuge."
```

Comment:
Very close here, but remember dot syntax in Ruby calls a method. If you want to access a property on a Hash you would use bracket syntax.

`'person.default = "#{person[:first_name]} is so cool. the best. yuge."'`